content/docs/guides/aws-lambda.md

```diff
@@ -146,11 +146,16 @@ Create the Lambda function using the [Serverless Framework](https://www.serverle
     'use strict';
 
     const { Client } = require('pg');
+    let client;
 
     module.exports.getAllUsers = async () => {
-     var client = new Client(process.env.DATABASE_URL);
-     client.connect();
-     var { rows } = await client.query('SELECT * from users');
+     if (!client) {
+       client = new Client(process.env.DATABASE_URL);
+       await client.connect();
```

Comment:
even if the client is present and being re-used, the connection still needs to be ensured using .connect(), correct?